libs/SalesforceSDKCore/SalesforceSDKCore/Classes/OAuth/SFOAuthCoordinator.m

```diff
@@ -217,55 +218,34 @@ - (void)authenticate {
         [self notifyDelegateOfBeginAuthentication];
         [self.oauthCoordinatorFlow beginJwtTokenExchangeFlow];
     } else {
-        __weak typeof(self) weakSelf = self;
-        switch (self.advancedAuthConfiguration) {
-            case SFOAuthAdvancedAuthConfigurationNone: {
-                [self notifyDelegateOfBeginAuthentication];
-                [self.oauthCoordinatorFlow beginUserAgentFlow];
-                break;
-            }
-            case SFOAuthAdvancedAuthConfigurationAllow: {
-
-                /*
-                 * If advanced auth mode is allowed, we have to get auth configuration settings
-                 * from the org, where available, and initiate advanced auth flows, if configured.
-                 */
-                [SFSDKAuthConfigUtil getMyDomainAuthConfig:^(SFOAuthOrgAuthConfiguration *authConfig, NSError *error) {
-                    __strong typeof(weakSelf) strongSelf = weakSelf;
+         __weak typeof(self) weakSelf = self;
+        if (self.requireCertificateAuthentication) {
```

Comment:
Is this the case where MDM specified that browser-based auth was required?

Reply:
@bhariharan Apologize. I wanted to drop some comments on the PR. This is meant to handle the mdm scenario.